src/v2i-hub/SpatPlugin/src/signalController.cpp

```diff
@@ -202,7 +190,7 @@ void SignalController::start_signalController()
 			//printf("Sleeping\n");
```

Comment:
Lets remove any commented code we think is no longer useful

Reply:
Will do

Reply:
addressed https://github.com/usdot-fhwa-OPS/V2X-Hub/pull/334/commits/dc6c71c5e57f2e367dd1db0cdeb0460870bfb01b